share/src/main/java/io/zensoft/share/service/Facebook/FacebookPublisherService.java

```diff
@@ -31,10 +33,14 @@
     private final Facebook facebookUser;
     private final Facebook facebookPage;
 
-    public FacebookPublisherService() throws Exception {
+    public FacebookPublisherService(){
         properties = new Properties();
```

Comment:
I think Spring provides annotations for reading and keeping key-value based properties